lemur/destinations/schemas.py

```diff
@@ -31,6 +31,9 @@ class DestinationOutputSchema(LemurOutputSchema):
     def fill_object(self, data):
         if data:
             data["plugin"]["pluginOptions"] = data["options"]
+            for option in data["plugin"]["pluginOptions"]:
+                if "export-plugin" in option["type"]:
+                    option["value"]["pluginOptions"] = option["value"]["plugin_options"]
```

Comment:
Function: This change is required to convert the `plugin_options` field (what's store in the DB) to the `PluginOptions` field that the Javascript expects. For the top level plugin, the options are stored as `options`; this inner plugin isn't actually converted to a plugin object and therefore just stays as `plugin_options` in the DB.

This feels rather hardcoded to me, but I am unsure what better options we have. Anyone more familiar with this is welcome to make suggestions.